example/src/main/tut/data/Maybe.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+title:  "Maybe"
+---
+
+# Maybe
+
+`Maybe` can either signal the presence or absence of a value and is a safer alternative to `scala.Option`
```

Comment:
Let's try to qualify statements like "is a safer alternative to [...]". For example, add

```
in the sense that it does not have unsafe methods like `Option#get`.
```